web/pdf_page_view.js

```diff
@@ -128,7 +128,15 @@ var PDFPageView = (function PDFPageViewClosure() {
         div.removeChild(node);
       }
       div.removeAttribute('data-loaded');
+      div.setAttribute('data-pageremoved-triggered', true);
```

Comment:
We don't need this attribute. The page renderingState will show if rendered page is present.